lib/isMobilePhone.js

```diff
@@ -34,7 +34,7 @@ var phones = {
   'en-HK': /^(\+?852\-?)?[456789]\d{3}\-?\d{4}$/,
   'en-IE': /^(\+?353|0)8[356789]\d{7}$/,
   'en-IN': /^(\+?91|0)?[6789]\d{9}$/,
-  'en-KE': /^(\+?254|0)?[7]\d{8}$/,
+  'en-KE': /^(\+?254|0)(7|1)\d{8}$/,
```

Comment:
Looks good. And also fixes a previous subtle bug that made `0/254` prefix optional when it should not be.